apps/web/package.json

```diff
@@ -22,6 +22,7 @@
     "@documenso/trpc": "*",
     "@documenso/ui": "*",
     "@hookform/resolvers": "^3.1.0",
+    "@react-pdf/renderer": "^3.3.8",
```

Comment:
we have `react-pdf` already, any reason for this?

Reply:
This is a different react-pdf 😄 This one is for creating PDF's in the declarative fashion

Reply:
precisely, this PR will be essentially creating PDFs documents.
this particular library is actually recommended by `react-pdf`: https://github.com/wojtekmaj/react-pdf?tab=readme-ov-file#lost